### ISSUE TITLE
Incorrect gas estimation

### ISSUE THREAD

**Issue by avinashbo** [sunnyside-io/op-erigon#183]
#### System information

Erigon version: `erigon version 0.05.2-unstable`

OS & Version: Linux/Docker

Commit hash: 

Erigon Command (with flags/config): `erigon --datadir=/home/node/data --db.pagesize=4096 --ethash.dagdir=/home/node/data/dag --db.size.limit=8TB --chain=base-mainnet --networkid=8453 --authrpc.jwtsecret=/home/node/jwt-secret.txt --authrpc.vhosts=* --http --http.addr=0.0.0.0`

Consensus Layer: NA

Consensus Layer Command (with flags/config): NA

Chain/Network:  Base and Optimism

#### Expected behaviour

Returns correct gas value

#### Actual behaviour

```
"code":-32000,"message":"insufficient funds for gas * price + value: 
```

#### Steps to reproduce the behaviour

```
curl "https://localhost:8545 " -X POST -H "Content-Type: application/json" -d  ' {"jsonrpc": "2.0", "id": "1", "method": "eth_estimateGas", "params": [{"from": "0xa5B8492D8223D255dB279C7c3ebdA34Be5eC9D85","data": "0x00", "to": "0xa5B8492D8223D255dB279C7c3ebdA34Be5eC9D85" }, "pending"]}'   
 ```

**Comment by mininny** [comment]
Hi, this issue is being tracked and fixed in upstream erigon in this issue: https://github.com/ledgerwatch/erigon/issues/10373. Once the fix is released as `2.60.1` in a short while, we will also make a followup release as `2.60.1-0.6.1`

**Comment by mininny** [comment]
Fixed in https://github.com/testinprod-io/op-erigon/releases/tag/v2.60.2-0.6.5. Thanks for your patience!
